lib/transaction/tracer/index.js

```diff
@@ -206,7 +200,7 @@ function _makeWrapped(tracer, handler, active, full) {
       logger.trace(err, 'Error from wrapped function:')
 
       if (prev === null && process.domain != null) {
```

Comment:
Don't we want to keep the `logger.trace(...)` part, though? That could be useful when debugging.

Reply:
So, as we determined, this is actually still being used, the segment will be fetched here:

https://github.com/newrelic/node-newrelic/blob/main/lib/instrumentation/core/domain.js#L17

Reply:
blah this domain code is wonky

Reply:
can you call this out in a comment?  so we avoid having this conversation in 6 months 😄

Reply:
Okay, after even more deliberation, I'm convinced that this should just be `symbols.segment` instead. It's only for possibly reporting the segment if it exists in case of an error message. This was accidentally removed in b8f219a9316bf0a97c3ee6492caa3254f2f5449b, where the only reference to read `__NR_transactionSegment` was replaced.